src/bootstrap/bin/rustc.rs

```diff
@@ -15,14 +15,14 @@
 //! switching compilers for the bootstrap and for build scripts will probably
 //! never get replaced.
 
-include!("../dylib_util.rs");
```

Comment:
I introduced `dylib_util.rs` to reduce compilation time by avoiding linking of libbootstrap and all it's dependencies when compiling the auxiliary helper executables.

Reply:
I am quite impressed from this hack :D Thank you for reminding.